docs/src/main/asciidoc/mongo-guide.adoc

```diff
@@ -0,0 +1,599 @@
+
+        try {
+            while (cursor.hasNext()) {
+                Document document = cursor.next();
+                Fruit fruit = new Fruit();
+                fruit.setName(document.getString("name"));
+                fruit.setDescription(document.getString("description"));
+                list.add(fruit);
+            }
+        } finally {
+            cursor.close();
+        }
+        return list;
+    }
+
+    public void add(Fruit fruit){
+        Document document = new Document()
+                .append("name", fruit.getName())
+                .append("description", fruit.getDescription());
+        getCollection().insertOne(document);
+    }
+
+    private MongoCollection getCollection(){
+        return mongoClient.getDatabase("fruit").getCollection("fruit");
+    }
+}
+----
+
+Now, edit the `org.acme.rest.json.FruitResource` class as follows:
+
+[source,java]
+----
+@Path("/fruits")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class FruitResource {
+
+    @Inject FruitService fruitService;
+
+    @GET
+    public List<Fruit> list() {
+        return fruitService.list();
+    }
+
+    @POST
+    public List<Fruit> add(Fruit fruit) {
+        fruitService.add(fruit);
+        return list();
+    }
+}
+----
+
+The implementation is pretty straightforward and you just need to define your endpoints using the JAX-RS annotations and use the FruitService to list/add new fruits.
+
+== Configuring the mongo database
+The main property to configure is the URL to access to MongoDB, almost all Mongo configuration can be put on the connection URI so we advise you to do so, you can find more information on the Mongo documentation site : https://docs.mongodb.com/manual/reference/connection-string/
+
+A sample configuration should look like this:
+[source,properties]
+----
+# configure mongo client for a replica set of two nodes
+quarkus.mongodb.connectionString = mongodb://mongo1:27017,mongo2:27017
+----
+
+If you need more configuration properties, there is a full list at the end of this guide.
+
+== Running a Mongo Database
+As by default, `MongoClient` is configure to access a local Mongo database on port 27017 (the default Mongo port), if you have a local running database on this port, there is nothing more to do before being able to test it!
+
+If you want to use Docker to run a Mongo database, you can use the following command to launch one:
+[source,shell]
+----
+docker run -ti --rm -p 27017:27017 mongo:4.0
+----
+
+== Creating a frontend
+
+Now let's add a simple web page to interact with our `FruitResource`.
+Quarkus automatically serves static resources located under the `META-INF/resources` directory.
+In the `src/main/resources/META-INF/resources` directory, add a `fruits.html` file with the content from this {quickstarts-blob-url}/rest-json/src/main/resources/META-INF/resources/fruits.html[fruits.html] file in it.
```

Comment:
I don't think you're referring to the right quickstart?

Reply:
it's copying the same file, but yes, would be better to provide the file in the right quickstart. Keeping this comment open until we have the right url.